aliasMultinomial.lua

```diff
@@ -0,0 +1,13 @@
+local AM = torch.class('torch.AliasMultinomial') 
```

Comment:
let's not expose this as a class like this.
Almost all math/RNG stuff in torch.* are exposed functionally.

If one wants a repeated draw, they should be able to call:
```
gen = torch.aliasmultinomial_generator(probs)
out = torch.aliasmultinomial(gen) -- or, out = torch.aliasmultinomial(out, gen)
```